sdk/entra/functions-authentication-events/review/functions-authentication-events.api.md

```diff
@@ -105,17 +108,17 @@ export interface FailedRequest extends AuthenticationEventResponse {
 
 // @public
 export interface ProvideClaimsForToken extends TokenIssuanceStartAction {
-    actionType: "ProvideClaimsForToken";
-    claims: TokenClaim[];
+    actionType: "microsoft.graph.ProvideClaimsForToken";
+    claims: TokenClaim;
 }
 
 // @public
 export type RequestStatus = "Failed" | "TokenInvalid" | "Successful";
 
 // @public
 export interface TokenClaim {
-    id: string;
-    value: string | string[];
+    // (undocumented)
+    [key: string]: string | string[];
```

Comment:
nit: It can read easier if this type is represented as 

```ts
export type TokenClaim = Record<string, string | string[]>
```

Reply:
Done